state/execution.go

```diff
@@ -266,17 +266,35 @@ func (blockExec *BlockExecutor) ApplyBlock(
 }
 
 func (blockExec *BlockExecutor) ExtendVote(vote *types.Vote) (types.VoteExtension, error) {
```

Comment:
There's inconsistency between the context's used here -- TODO and Background. Also, no need to place them into a variable if they're only used/passed directly into the API call.